contracts/tec/contracts/src/libs/LibZeroExTransaction.sol

```diff
@@ -0,0 +1,91 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.3;
+
+import "./LibEIP712Domain.sol";
+
+
+contract LibZeroExTransaction is
+    LibEIP712Domain
+{
+    // Hash for the EIP712 0x transaction schema
+    bytes32 constant internal EIP712_ZEROEX_TRANSACTION_SCHEMA_HASH = keccak256(abi.encodePacked(
```

Comment:
We could save a few hundred gas per call by hardcoding this; the hash (counter-intuitively) gets executed on each call - even though it’s `constant`.

Reply:
Added a task to do this later, there are definitely a good amount of optimizations to be done.